pkg/lint/rules/testdata/albatross/templates/svc.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   labels:
     app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
     app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
```

Comment:
The `Chart.yaml` should include a version if it's going to be called here. We should either omit the `AppVersion` or add it to the `Chart.yaml`